pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -497,13 +495,19 @@ def _validate_monitor_key(self, trainer):
             )
             raise MisconfigurationException(m)
 
-    def _get_metric_interpolated_filepath_name(self, ckpt_name_metrics: Dict[str, Any], epoch: int, step: int):
+    def _get_metric_interpolated_filepath_name(
+        self, ckpt_name_metrics: Dict[str, Any], epoch: int, step: int, del_list: List[str] = []
```

Comment:
Can we rename `del_list` to something clearer like `existing_filepath`? I understand you are reusing the name in the `_update_best_and_save` method but the usage here is unrelated to the fact that it will be deleted later

Reply:
changed it to del_filepath. I believe it self explains that this file will be deleted so no need to update.